src/oop/programs_with_multiple_classes/medium/q2/MainTest.java

```diff
@@ -0,0 +1,163 @@
+package oop.programs_with_multiple_classes.medium.q2;
+
+import global.BaseTest;
+import global.ObjectTest;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.IntegerLiteral;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import global.variables.wrappers.Optional;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static global.tools.CustomAssertions._assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class MainTest extends BaseTest {
+    // Parsons
+    public ObjectTest bed;
+    public ObjectTest pillow;
+    public ObjectTest testRoom;
+
+    @BeforeEach
+    public void setup() {
+        String bedClassString = "oop.programs_with_multiple_classes.medium.q2.Bed";
+        String pillowClassString = "oop.programs_with_multiple_classes.medium.q2.Pillow";
+        String testRoomClassString = "oop.programs_with_multiple_classes.medium.q2.TestRoom";
+        bed = new ObjectTest(bedClassString);
+        pillow = new ObjectTest(pillowClassString);
+        testRoom = new ObjectTest(testRoomClassString);
```

Comment:
Test room has a main method, so you should just test it using BaseTest, you don't also need an ObjectTest